26.2.0 GA release notes (#23205)
cockroachdb/docs: src/current/releases/cloud.md

{% include marketo.html formId=1083 %}

## April 28, 2026

<div id="feature-highlights">

<table>
 <thead>
  <tr>
   <th class="center-align">Feature</th>
   <th class="center-align">Availability</th>
   <th style="white-space: nowrap;">Self-hosted</th>
   <th>Basic</th>
   <th>Standard</th>
   <th>Advanced</th>
  </tr>
 </thead>
 <tbody>
  <tr>
   <td>
    <p class="feature-summary">Active-passive deployment across two data centers</p>

>     <p class="feature-description">CockroachDB Advanced clusters can be deployed across two geographically separated data centers in an active-passive configuration, providing resilience against data center failures.</p>

Reviewer: Should be removed - this was bumped to Q2 https://cockroachlabs.atlassian.net/browse/DELI-33

   </td>
   <td class="icon-center">Preview</td>
   <td class="icon-center">{% include icon-no.html %}</td>
   <td class="icon-center">{% include icon-no.html %}</td>
   <td class="icon-center">{% include icon-no.html %}</td>
   <td class="icon-center">{% include icon-yes.html %}</td>
  </tr>
  <tr>
   <td>
    <p class="feature-summary">CockroachDB Cloud CLI</p>

>     <p class="feature-description">The CockroachDB Cloud CLI has been redesigned with updated commands for managing clusters, users, and cloud resources from the terminal.</p>

Reviewer: Link to https://www.cockroachlabs.com/docs/cockroachcloud/ccloud-get-started

   </td>
   <td class="icon-center">GA</td>
   <td class="icon-center">{% include icon-no.html %}</td>
   <td class="icon-center">{% include icon-yes.html %}</td>
   <td class="icon-center">{% include icon-yes.html %}</td>
   <td class="icon-center">{% include icon-yes.html %}</td>
  </tr>
  <tr>
   <td>

>     <p class="feature-summary">Multi-factor authentication</p>

Reviewer 2: ```suggestion
    <p class="feature-summary">Multi-factor authentication enforcement for the Cloud Console</p>
```

>     <p class="feature-summary">Multi-factor authentication</p>

Reviewer 2: Currently trying to figure out if this is even making it in. Will update this.

Author: Removed per your update!

>     <p class="feature-description">Multi-factor authentication is now enforced for all CockroachDB Cloud users, reducing the risk of unauthorized access from compromised credentials.</p>

Reviewer 2: ```suggestion
    <p class="feature-description">Multi-factor authentication can now be enforced for CockroachDB Cloud users who access the Console via username and password, reducing the risk of unauthorized access from compromised credentials.</p>
```

   </td>

>    <td class="icon-center">GA</td>

Reviewer 2: ```suggestion
   <td class="icon-center">Preview</td>
```

   <td class="icon-center">{% include icon-no.html %}</td>
   <td class="icon-center">{% include icon-yes.html %}</td>
   <td class="icon-center">{% include icon-yes.html %}</td>
   <td class="icon-center">{% include icon-yes.html %}</td>
  </tr>
  <tr>
   <td>

>     <p class="feature-summary">Bring your own cloud (BYOC)</p>

Reviewer: ```suggestion
    <p class="feature-summary">Bring Your Own Cloud (BYOC)</p>
```

>     <p class="feature-description">CockroachDB Cloud clusters can run within your own AWS, Azure, or GCP account. This gives you control over networking, security, and data residency while retaining managed database operations.</p>

Reviewer: Link to https://www.cockroachlabs.com/docs/cockroachcloud/byoc-overview

>     <p class="feature-description">CockroachDB Cloud clusters can run within your own AWS, Azure, or GCP account. This gives you control over networking, security, and data residency while retaining managed database operations.</p>

Reviewer: ```suggestion
    <p class="feature-description">You can deploy CockroachDB Cloud clusters within your own AWS, Azure, or GCP account. This gives you access to your existing cloud savings and control over networking, security, and data residency while retaining the managed database operations of CockroachDB Cloud.</p>
```

Reviewer: Should run this by Lakshmi as well. I can do that if you'd like, just let me know

   </td>
   <td class="icon-center">Preview</td>
   <td class="icon-center">{% include icon-no.html %}</td>
   <td class="icon-center">{% include icon-no.html %}</td>
   <td class="icon-center">{% include icon-no.html %}</td>
   <td class="icon-center">{% include icon-yes.html %}</td>
  </tr>
  <tr>
   <td>
    <p class="feature-summary">Model Context Protocol (MCP) server</p>

>     <p class="feature-description">The CockroachDB Cloud MCP server allows AI agents and LLM-powered applications to connect to CockroachDB using the Model Context Protocol (MCP).</p>

Reviewer 2: ```suggestion
    <p class="feature-description">The CockroachDB Cloud MCP server allows your AI agents and LLM-powered applications to connect to CockroachDB using the Model Context Protocol (MCP).</p>
```

Legal wants us to use "your".

>     <p class="feature-description">The CockroachDB Cloud MCP server allows AI agents and LLM-powered applications to connect to CockroachDB using the Model Context Protocol (MCP).</p>

Reviewer 2: Link to [https://www.cockroachlabs.com/docs/cockroachcloud/connect-to-the-cockroachdb-cloud-mcp-server](https://www.cockroachlabs.com/docs/cockroachcloud/connect-to-the-cockroachdb-cloud-mcp-server)

   </td>
   <td class="icon-center">GA</td>
   <td class="icon-center">{% include icon-no.html %}</td>
   <td class="icon-center">{% include icon-yes.html %}</td>
   <td class="icon-center">{% include icon-yes.html %}</td>
   <td class="icon-center">{% include icon-yes.html %}</td>
  </tr>
  <tr>
   <td>
    <p class="feature-summary">Fault tolerance demo</p>

>     <p class="feature-description">The built-in <a href="{% link {{ site.versions["stable"] }}/demo-cockroachdb-resilience.md %}#run-a-guided-demo-in-cockroachdb-cloud">fault tolerance demo</a> is now generally available. This demo simulates an availability zone failure in a live cluster, allowing you to watch the cluster automatically recover as traffic continues uninterrupted.</p>

Reviewer: ```suggestion
    <p class="feature-summary">Fault tolerance demo</p>
    <p class="feature-description">Run a built-in <a href="{% link {{ site.versions["stable"] }}/demo-cockroachdb-resilience.md %}#run-a-guided-demo-in-cockroachdb-cloud>fault tolerance demo</a> in the CockroachDB Cloud Console that allows you to monitor query execution during a simulated failure and recovery.</p>
```

Reviewer: Confirmed with Ayushi and Lauren that we are _not_ moving the fault tolerance demo to GA with this release. Replacing this text with the existing release note that we had added

   </td>
   <td class="icon-center">GA</td>
   <td class="icon-center">{% include icon-no.html %}</td>
   <td class="icon-center">{% include icon-yes.html %}</td>
   <td class="icon-center">{% include icon-yes.html %}</td>
   <td class="icon-center">{% include icon-yes.html %}</td>
  </tr>
 </tbody>
</table>

</div>

## Feb 24, 2026

> CockroachDB {{ site.data.products.cloud }} {{ site.data.products.advanced }} users can now run a built-in [fault tolerance demo]({% link {{ site.versions["stable"] }}/demo-cockroachdb-resilience.md %}#run-a-guided-demo-in-cockroachdb-cloud) that allows you to monitor query execution during a simulated failure and recovery. The fault tolerance demo is in [Preview]({% link {{ site.versions["stable"] }}/cockroachdb-feature-availability.md %}).

Reviewer: Since I repurposed this release note for the feature highlight, we should consider removing this release note as a duplicate. I'm not pressed about keeping the release note around given previous conversations about removing individual Cloud release notes altogether.
